browser-test/login.test.js

```diff
@@ -0,0 +1,63 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {getDriver, getElementByXPath, urls, initialData} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    jest.setTimeout(30000);
+    driver = await getDriver(Builder, new firefox.Options().headless());
+  });
+
+  afterAll(async () => {
+    driver.close();
```

Comment:
I think we will need a clean up python script to be called when tests are finished, to clear up any data which is created by the tests